src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -68,13 +75,26 @@ internal OtlpExporterOptions(
 
         this.DefaultHttpClientFactory = () =>
         {
+#if NET6_0_OR_GREATER
+            var handler = new HttpClientHandler();
+            HttpClient client = this.AddCertificatesToHttpClient(handler);
+            client.Timeout = TimeSpan.FromMilliseconds(this.TimeoutMilliseconds);
+            return client;
+#else
+            // For earlier .NET versions
```

Comment:
This is incorrect; only the latest .NET versions use `HttpClient`. .NET Framework and older versions of .NET Core still rely on `Grpc.Core`.